scripts/build/build-parser

```diff
@@ -11,13 +11,19 @@ npm i
 npm run build-parser
 npm run build-wasm-bindings
 
-# copy JS/WASM bindings to backend/static/tree-sitter
+
+# make parser binary
+gcc -o tree-sitter-darklang.so -shared src/parser.c -Isrc -O3 -fPIC
+
 cd ../..
-mkdir -p backend/static/tree-sitter
-cp parser/tree-sitter-darklang/bindings/tree-sitter-darklang.wasm backend/static/tree-sitter/
-cp parser/tree-sitter-darklang/node_modules/web-tree-sitter/tree-sitter.js backend/static/tree-sitter/
-cp parser/tree-sitter-darklang/node_modules/web-tree-sitter/tree-sitter.wasm backend/static/tree-sitter/
 
-# also copy the WASM bindings to vscode-extension/static
+
+# copy the WASM bindings to vscode-extension/static
 mkdir -p vscode-extension/static/tree-sitter
-cp parser/tree-sitter-darklang/bindings/tree-sitter-darklang.wasm vscode-extension/static/tree-sitter/
\ No newline at end of file
+cp parser/tree-sitter-darklang/bindings/tree-sitter-darklang.wasm vscode-extension/static/tree-sitter/
+
+# copy .so files for tree-sitter and tree-sitter-darklang
+#   to corresponding .NET projects
+printf "!! do this manually !!\n  get the platform-dependent tree-sitter.so file from\n  https://github.com/davidelettieri/treesitter-bindings/raw/main/tree-sitter-packages/libtreesitter.runtime.linux-x64/libtree-sitter.so\n  and put it in backend/src/LibTreeSitter/\n  , renamed as tree-sitter.so\n"
```

Comment:
ugly

Reply:
some options:
- just include the .so in our code
- ...

actually, I'll just do that - it's only 200KB, and changes rarely - not sure why I was avoiding that.